src/StoreApi/Routes/ProductCategories.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Product Categories route.
+ *
+ * @internal This API is used internally by Blocks--it is still in flux and may be subject to revisions.
+ * @package WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\StoreApi\Routes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * ProductCategories class.
+ */
+class ProductCategories extends AbstractTermsRoute {
```

Comment:
This class is not implementing the count property right.

in Master:
<img width="763" alt="Screen Shot 2020-04-24 at 12 53 14 PM" src="https://user-images.githubusercontent.com/6165348/80210240-579f6100-862b-11ea-8d4c-fe8c901941a8.png">

In this branch:
<img width="762" alt="Screen Shot 2020-04-24 at 12 32 33 PM" src="https://user-images.githubusercontent.com/6165348/80210246-59692480-862b-11ea-9dd9-e2b2b4d6c471.png">